src/TheOpenNetwork/Address.cpp

```diff
@@ -103,10 +97,6 @@ Address::Address(const std::string& string) {
     }
 }
 
-Address::Address(const PublicKey& publicKey, int8_t workchainId)
```

Comment:
[It is told ](https://developer.trustwallet.com/wallet-core/newblockchain#address-encoding-decoding) that "Address" should have "a constructor from a public key", but in TON we usually operate with a different entity called "Wallet", and the address is derived from it. 

Could you confirm if we actually should add this constructor anyway?
After this commit, the wallet should be created with a public key, and then the address can be calculated.

Reply:
Ok👍